src/alt.js

```diff
@@ -34,6 +34,13 @@ function uid(container, name) {
   return key
 }
 
+/* for now just detect whether it's instance of Immutable.js */
+function isImmutable(state) {
+  // let's check with that for now
+  // https://github.com/facebook/immutable-js/issues/421
+  return typeof state.toJS === 'function'
```

Comment:
@goatslacker makes sense to me. I just always like to play devil's advocate to adding things to make sure we thing about them :smiling_imp: